src/containers/MyNdla/components/NavigationLink.tsx

```diff
@@ -47,15 +51,38 @@ const IconWrapper = styled.span`
   justify-content: center;
 `;
 
+const LongText = styled.span`
+  ${mq.range({ from: breakpoints.tablet, until: breakpoints.desktop })} {
+    display: none;
+    width: 0px;
+  }
+`;
+
+const ShortText = styled.span`
+  ${mq.range({ from: breakpoints.desktop })} {
+    display: none;
+  }
+  ${mq.range({ from: breakpoints.mobile, until: breakpoints.tablet })} {
+    display: none;
+  }
+`;
```

Comment:
Dette blir bare midlertidig løsning til vi går over til hovedMenyen. Da blir dette uansett fjernet siden vi ikke trenger en standalone side for mobil meny

Reply:
I framtiden skal det kun være en mq.range og ikke  to